packages/integrations/components/coral/withPico.js

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import CoralEmbed from './index';
+import {
+  actionVerifyPicoUser,
+  actionReceiveCoralLogoutRequest,
+} from '../../actions';
+import { tokenSelector } from '../../selectors/coralSelector';
+
+const withPico = (ChildComponent) => (props) => {
+  const [verificationRequestSent, setRequestStatus] = useState(false);
```

Comment:
I haven't used the new React hooks myself much, but I take it using them here is justifiable since it's simpler snd there's no real scenario where components elsewhere in the app would need to concurrently look at the Pico state? Would there be a scenario where you'd want this in the global Redux store instead of with `useState`?

Reply:
@xmattus I don't believe so. The state value being set here is more of a flag to tell the component that a verification request has already been sent and to not execute another if the `MutationObserver` fires more than once, which I did experience during the development process.

Reply:
Sounds good to me!